packages/flutter/lib/src/painting/image_stream.dart

```diff
@@ -587,6 +600,20 @@ abstract class ImageStreamCompleter with Diagnosticable {
     }
   }
 
+  /// Remove a listener for errors.
+  /// This is similar to [removeListener]. However, even if all listeners have
+  /// been removed and all [keepAlive] handles have been disposed, this image
+  /// stream will not be automatically disposed.
```

Comment:
I am not sure what this "however" sentence is supposed to tell me. I expected an explanation of how this is different from removeListener, but removeListener (or anything else on this class) doesn't document anything about automatic disposal of the image stream. I am not even sure what is meant by automatic disposal...